lua/smh/server/positioning.lua

```diff
@@ -29,6 +29,10 @@ function SMH.GetPositionFrames(frames, framepos, ignoreCurrentFrame)
 
 	for _, frame in pairs(frames) do
 
+		if frame.Position == nil then
```

Comment:
This check should not be necessary. Is there some error happening if it's not here?

Reply:
Yeah, it seemed to cause the issue with duplicating a frame and pasting it onto itself with ctrl + middle mouse or what was the button to delete a frame, according to what was said on the steam workshop fix.